thesis/03-implementation.tex

```diff
@@ -175,24 +174,24 @@ \subsection{Components overview}
 
 \subsubsection{Frontend}
 
-The web application contains two main subcomponents: \textit{Guide Creator} and \textit{Guide Viewer}, responsible respectively for: creating instructions and viewing animated guides. Other pages are mainly related to the community aspect and will only be covered superficially in this section.
+The web application contains two main subcomponents: \textit{Guide Creator} and \textit{Guide Viewer}, responsible respectively for creating instructions, and viewing animated guides. Other components are mainly related to the community aspect and will only be covered superficially.
```

Comment:
```suggestion
The web application contains two main subcomponents: \textit{Guide Creator} and \textit{Guide Viewer}, respectively responsible for creating instructions and viewing animated guides. Other components are mainly related to the community aspect and will only be covered superficially.
```
Not sure if `responsible respectively` is correct, but quick ducky search highly favors `respectively responsible for` to `responsible respectively`.

I think the comma before `and` might be redundant or even incorrect, because we have only two items in the list. Some examples and discussion: https://ell.stackexchange.com/questions/32213/where-to-put-respectively